packages/split-layout/src/vaadin-split-layout.js

```diff
@@ -327,23 +327,15 @@ class SplitLayout extends ElementMixin(ThemableMixin(mixinBehaviors([IronResizab
   }
 
   /**
-   * Can be called to manually notify a resizable and its descendant
-   * resizables of a resize change.
+   * @deprecated Since Vaadin 23, `notifyResize()` is deprecated. The component uses a
+   * ResizeObserver internally and doesn't need to be explicitly notified of resizes.
```

Comment:
I wonder if we should have `NotifyResizeMixin` to have this method there to avoid copy-paste 🤔 
Not a blocker, just a random idea which came to my mind while looking through these PRs.

Reply:
We could add it if it's worth it. The deprecated functions will anyways be removed right after V23.